rero_ils/modules/collections/views.py

```diff
@@ -60,11 +60,12 @@ def collection_view_method(pid, record, template=None, **kwargs):
         item['document'] = Document.get_record_by_pid(
```

Comment:
I do not think you need the line #51 `libraries = []` anymore

Reply:
This array is used at line 65